src/makeRobot.js

```diff
@@ -38,7 +38,46 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+        + ` wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(step = 1) {
```

Comment:
```suggestion
    get info() {
      return `name: ${this.name}, chip version: ${this.version},`
        + ` wheels: ${this.wheels}`;
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    goForward(step = 1) {
```
Don't forget to add newlines between methods to make your code easier to read